 	float32[] x_position
 	float32[] y_position
 	float32[] z_position
 	float32[] tf
	int16[] angle
	uint8[] aruco_id
	uint16 visible_aruco